docs/en/writing-running-appium/ios/ios-xctest-file-movement.md

```diff
@@ -25,13 +25,13 @@ $ brew install ifuse --HEAD # newer iOS versions need the latest codebase
 
 The format of method argument should be the following:
 
-- `@<app_bundle_id>` is a bundle id
-- `optional_container_type` is container type
-    - `documents` is only available
-        - BundleIds which can get by `ifuse -u <udid> --list-apps` can specify `documents` type
+- `@<app_bundle_id>` is the application bundle identifier
+- `optional_container_type` is the container type
+    - `documents` is the only available option
+        - You may specify `documents` container type only for bundle ids returned by `ifuse -u <udid> --list-apps`
         - e.g. Below _On My iPhone_ image has _Slack_ folder, but `com.tinyspeck.chatlyio` does not exist in the output of `--list-apps`. Thus, we cannot mount it as `com.tinyspeck.chatlyio@documents/appium.png`
```

Comment:
github markdown allows to inline images:

https://guides.github.com/features/mastering-markdown/